sockeye/training.py

```diff
@@ -108,6 +108,8 @@ def _build_module(self, train_iter: data_io.BaseParallelSampleIter):
         target_length = utils.compute_lengths(target)
         labels = mx.sym.reshape(data=mx.sym.Variable(C.TARGET_LABEL_NAME), shape=(-1,))
 
+        source_factors = [mx.sym.Variable(C.SOURCE_NAME + "_factor_" + str(i)) for i in range(self.config.num_factors)]
```

Comment:
lets move this above line 106.
also use format string.

Reply:
ok